src/RangeEnclosures.jl

```diff
@@ -27,4 +27,6 @@ include("enclose.jl")
 export enclose,
   NaturalEnclosure, MooreSkelboeEnclosure, SumOfSquaresEnclosure, TaylorModelsEnclosure
 
+const available_solvers = (NaturalEnclosure, MooreSkelboeEnclosure,
+                           SumOfSquaresEnclosure, TaylorModelsEnclosure)
```

Comment:
Can this be used for anything? I would rather advise to do `subtypes(AbstractEnclosureAlgorithm)`, which is less fragile and will also work for user-defined extensions.

Reply:
good point! I'll remove this and update the docstring in `enclose`